mvtprovider/postgis/README.md

```diff
@@ -1,32 +1,22 @@
 # PostGIS MVT Provider
 
-The PostGIS MVT provider manages querying for tile requests against a Postgres database (version 12+) with the [PostGIS](http://postgis.net/)(version 3.0+) extension installed and leverages [ST_AsMVT](https://postgis.net/docs/ST_AsMVT.html) to handle the MVT encoding at the database. 
+The PostGIS MVT provider manages querying for tile requests against a Postgres database (version 12+) with the [PostGIS](http://postgis.net/)(version 3.0+) extension installed and leverages [ST_AsMVT](https://postgis.net/docs/ST_AsMVT.html) to handle the MVT encoding at the database.
 
 The connection between tegola and PostGIS is configured in a `tegola.toml` file. An example minimum connection config:
 
-
 ```toml
 [[providers]]
 name = "test_postgis"       # provider name is referenced from map layers (required)
 type = "mvt_postgis"        # the type of data provider must be "mvt_postgis" for this data provider (required)
-host = "localhost"          # PostGIS database host (required)
-port = 5432                 # PostGIS database port (required)
-database = "tegola"         # PostGIS database name (required)
-user = "tegola"             # PostGIS database user (required)
-password = ""               # PostGIS database password (required)
+uri = "postgres://postgres:postgres@localhost:5432/tegola"          # PostGIS database uri (required)
 ```
 
 ### Connection Properties
 
-- `name` (string): [Required] provider name is referenced from map layers.
-- `type` (string): [Required] the type of data provider. must be "postgis" to use this data provider
-- `host` (string): [Required] PostGIS database host
-- `port` (int): [Required] PostGIS database port (required)
-- `database` (string): [Required] PostGIS database name
-- `user` (string): [Required] PostGIS database user
-- `password` (string): [Required] PostGIS database password
-- `max_connections` (int): [Optional] The max connections to maintain in the connection pool. Defaults to 100. 0 means no max.
```

Comment:
We should add a reference around some of these pgx specific configs. It looks like the [`ParseConfig` ](https://pkg.go.dev/github.com/jackc/pgx/v4/pgxpool#ParseConfig) function for pgxpool supports the following additional query strings:

* pool_max_conns: integer greater than 0 
* pool_min_conns: integer 0 or greater 
* pool_max_conn_lifetime: duration string 
* pool_max_conn_idle_time: duration string 
* pool_health_check_period: duration string 
* pool_max_conn_lifetime_jitter: duration string

I think adding some notes to the README about how this package uses `pgx` under the hood with a link with to the ParseConfig function and indicating some of the available connection config query strings would be really helpful.